package.json

```diff
@@ -8,7 +8,8 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "nodemon server/app.js"
+    "start": "nodemon server/app.js",
+    "test": "jest --detectOpenHandles"
   },
   "dependencies": {
     "body-parser": "^1.18.3",
```

Comment:
Hey Gary,

Express -v > 4 comes with body-parser Json and urlencoded already integrated into it's framework.  Take a look at [this page from Express](https://expressjs.com/en/4x/api.html).  There is no need with the version of Express to incorporate body-parser into your project since it's already there for the two most common parser functions.

Unless you are interested in using the [Body-Parser](https://www.npmjs.com/package/body-parser) 'Text' or 'Raw' functions, i'd remove the package.

Reply:
Thanks Grahn! I keep npm installing body-parser out of habit. I'll remove the package and update my code accordingly.